### PR TITLE
module: add more module/global options

### PR DESCRIPTION
**Pull request by lasers**
Addresses #1078.

### CODE REVIEW

**Comment by tobes** [comment]
hey @laser can you split this

you have

* unrelated documentation changes
* changes to default settings
* changes to module.py current functionality
* added features

These all need to be done separately

**Comment by lasers** [comment]
@tobes You got it, pal. :+1: I wait for other PRs to be merged first before I rebase this.

File: doc/configuration.rst [hunk 1]
```diff
@@ -71,8 +71,7 @@ overridden by also defining in the individual module.
 
 Global options:
 
-- ``nagbar_font``. It will be used as an argument to
-    ``i3-nagbar -f``, thus setting its font.
+``nagbar_font``. Specify a font for ``i3-nagbar -f <font>``.
```
Comment (tobes) [inline doc/configuration.rst]: this is unrelated

File: doc/configuration.rst [hunk 2]
```diff
@@ -93,7 +92,7 @@ Store cache in $XDG_CACHE_HOME or ~/.cache
 
     # default behavior
     py3status {
-        storage =  'py3status_cache.data'
+        storage = 'py3status_cache.data'
```
Comment (tobes) [inline doc/configuration.rst]: this is unrelated

File: doc/configuration.rst [hunk 3]
```diff
@@ -294,7 +330,7 @@ You can:
          format = "MPD: {state} {artist} {title}"
       }
 
-  - Unknown placeholders act as if they were static text and placeholders that are empty or None will be removed. 
+  - Unknown placeholders act as if they were static text and placeholders that are empty or None will be removed.
```
Comment (tobes) [inline doc/configuration.rst]: this is unrelated

**Comment by ultrabug** [comment]
ping @lasers closing this one since there's too much in it, would be cool to propose only the alpha thing on a simple PR!

cheers

**Comment by lasers** [comment]
I don't have alpha thing working. This PR would need to be fixed either way after merging #1486. 

#1486 works okay. I made temp `get_color` function instead of importing `Py3`'s `get_color()`.
